variables.tf

```diff
@@ -121,19 +123,20 @@ variable "bucket_location" {
 
 variable "auto_create_network" {
   description = "Create the default network"
-  default     = "false"
+  type = bool
+  default     = false
 }
 
 variable "lien" {
   description = "Add a lien on the project to prevent accidental deletion"
-  default     = "false"
-  type        = string
+  default     = false
```

Comment:
Not related to your change, please revert

Reply:
Reverted